x-pack/plugin/searchable-snapshots/src/test/resources/rest-api-spec/api/searchable_snapshots.stats.json

```diff
@@ -0,0 +1,30 @@
+{
+  "searchable_snapshots.stats": {
+    "documentation": {
+      "url": "https://www.elastic.co/guide/en/elasticsearch/reference/current/searchable-snapshots-get-stats.html //NORELEASE"
+    },
+    "stability": "experimental",
```

Comment:
We should at least mark this line as `//NORELEASE` too, but really I think we should be bolder and say that by the time this is merged we expect this API not to be experimental any more, so there's no need for this.

Reply:
I agree. Sadly we can't mark this line as `//NORELEASE` too as the `stability` is checked against a specified set of values. We can't also comment this JSON file.

I think that `experimental` reflects the current state of this API for now. I hooked on the //NORELEASE and add more documentation about the stability of the API (see fde0a86e63f0ba22ff6aa1d5d29ddb5b5b1b49ec).